csv_detective/explore_csv.py

```diff
@@ -373,6 +380,7 @@ def routine_minio(
 
     generate_table_schema(
         return_dict,
+        True,
```

Comment:
(Not about this commit but)
lines 314-325: "and" & "or" statements without brackets, intended?